logging_config.ini

```diff
@@ -39,5 +39,15 @@ level=WARNING
 formatter=customFormatter
 args=('/dev/log',)
 
+# standart output handler
+[handler_stdoHandler]
+class=StreamHandler
+level=DEBUG
```

Comment:
I would put this to `WARNING` by default, otherwise the output is extremely verbose.
What do you think?

Reply:
@arbulu89  my reason is that I expect that normally the exporter run as daemon and nobody will run it as CLI. So the only ones who run it not as daemon will be devs so that's why i put the debug there..